flink-python/pyflink/datastream/connectors/tests/test_pulsar.py

```diff
@@ -253,7 +250,6 @@ def test_pulsar_sink(self):
                 .long_type()
                 .no_default_value()._j_config_option), 100)
 
-    @unittest.skip("Should be moved to pulsar connector repo")
```

Comment:
How are all these python changes related to the PR? Should we maybe remove them from here?

Reply:
Given the rework there is no need to disable the tests anymore. The previous implementation broke these tests due to the connector externalization work not being fully finished for the Python language binding of the connectors.

Reply:
Thanks @mbalassi for answering the question.
This is exactly the situation. This is a revert of disabling the python tests